src/Gui/Utilities.h

```diff
@@ -441,6 +441,30 @@ class GuiExport ItemViewSelection
     class MatchName;
 };
 
+#define FC_ADD_ENTRY(__part__, __partclass__, __parent__) SO_KIT_ADD_CATALOG_ENTRY(__part__, __partclass__, TRUE, __parent__, "", TRUE);

```

Comment:
`FC_ADD_CATALOG_ENTRY`? The `CATALOG` here is very important for the NodeKit terminology.

Reply:
Makes sense. I will update the code accordingly.

Reply:
fixed